zone/corpse.cpp

```diff
@@ -647,6 +660,29 @@ void Corpse::DepopPlayerCorpse() {
 	player_corpse_depop = true;
 }
 
+void Corpse::AddConsentName(const char* add_name)
+{
+	if (add_name) {
+		for (const auto& n : consent_names) {
+			if (strcasecmp(n.c_str(), add_name) == 0) {
+				return;
+			}
+		}
+		consent_names.emplace_back(add_name);
+	}
+}
+
+void Corpse::RemoveConsentName(const char* rem_name)
```

Comment:
Same thing here, is there a reason we didn't just use `std::string consent_player_name`